### PR TITLE
Attempt to restore manylinux1 image build and publish

### PR DESCRIPTION
**Pull request by bretambrose**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .github/docker-images/manylinux1-x64/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# See: https://github.com/pypa/manylinux	FROM docker.pkg.github.com/awslabs/aws-crt-python/manylinux1:x64
+# and: https://github.com/pypa/python-manylinux-demo
+FROM quay.io/pypa/manylinux1_x86_64
```
Comment (TwistedTwigleg) [inline .github/docker-images/manylinux1-x64/Dockerfile:3]: Will manylinux1 being not actively supported be an issue? From what I can tell, it seems to mostly apply to using Python 3.10+, so my understanding is that so long as we do not need Python 3.10+ it should be good?

Comment (TwistedTwigleg) [reply .github/docker-images/manylinux1-x64/Dockerfile:3]: Did some additional reading and from what I can tell reading the issue on dropping mannylinux1 support on the MannyLinux repository, it looks like it is still getting automatic updates and builds. It seems that as long as we do not need the latest Python and Pip versions for this particular DockerFile, it should be fine.

Comment (TingDaoK) [reply .github/docker-images/manylinux1-x64/Dockerfile:3]: Yeah, it doesn't have python 3.10. We are not testing python 3.10, so, we are good. And, we will drop it as soon as CLI drops the support.

**Review by TingDaoK — APPROVED** [review]
🙆‍♂️

File: .github/docker-images/manylinux1-x64/Dockerfile (hunk 1 above)
Comment (TwistedTwigleg) [reply .github/docker-images/manylinux1-x64/Dockerfile:3]: Ah okay, good to know - Thanks!

**Pull request merged**